plasmapy/physics/parameters.py

```diff
@@ -1116,11 +1135,150 @@ def Debye_number(T_e, n_e):
     """
 
     lambda_D = Debye_length(T_e, n_e)
+
     N_D = (4 / 3) * np.pi * n_e * lambda_D ** 3
 
     return N_D.to(u.dimensionless_unscaled)
 
 
+@utils.check_quantity({
+    'V_0': {'units': u.V, 'can_be_negative': False},
+    'n_s': {'units': u.m ** -3, 'can_be_negative': False}
+    })
+def matrix_sheath_thickness(V_0, n_s):
+    r"""Calculate the matrix sheath thickness for a high potential sheath.
```

Comment:
I think this could potentially be more descriptive than repeating the function's name, but I don't really have any better suggestions!